source/apple-pay-and-google-pay.html.erb

```diff
@@ -25,10 +25,10 @@ title: Apple Pay and Google Pay
           on GOV.UK&nbsp;Pay.
         </p>
 
-        <h2 class="govuk-heading-m">How Apple Pay and Google Pay works</h2>
+        <h2 class="govuk-heading-m">How Apple Pay and Google Pay work</h2>
 
         <p class="govuk-body">Any user with Apple Pay or Google Pay enabled can make payments on your service in seconds.</p>
-        <p class="govuk-body">Because Apple Pay and Google already have users details, they don’t have to fill them in again on GOV.UK&nbsp;Pay. They simply have to:</p>
+        <p class="govuk-body">Because Apple Pay and Google already have users' details, they do not have to fill them in again on GOV.UK&nbsp;Pay. They just have to:</p>
```

Comment:
Can this apostrophe (and the others) be curly — so `’` rather than `'`?

Reply:
I've hopefully fixed that now. Is that everything?